docs/requirements.txt

```diff
@@ -1,2 +1,2 @@
-sphinx==5.3.0
+sphinx
 sphinx-autoapi
```

Comment:
No need to add `sphinx_rtd_theme` ?

Reply:
I wanted to see if it worked without it, but since it didn't, I added it. :)

Reply:
RTD injects the needed packages apparently:

python -m pip install --upgrade --no-cache-dir sphinx readthedocs-sphinx-ext